ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1668,6 +1669,41 @@ daemons:
 
 This will add a dameon to the eos configuration that is most useful when trying to run OpenConfig clients like ocprometheus
 
+#### Connectivity Monitor
+
+```yaml
+monitor_connectivity:
+    shutdown: < true | false >
+    interval: < probing_interval >
+    interface_sets:
+      - name: < interface_set >
+# Interfaces should be of same type, Ethernet, Loopback, Management etc.
+        interfaces:
```

Comment:
I suggest to simplify the model to just use a string on `interfaces` instead of a list of strings. This would also be closer to eos, since it only accept a single range of a single type. (keep the comment). Also add quotes to ensure the user understands it is a string.
`interfaces: "< interface_or_interface_range >"`.

Reply:
It actually accepts this way Ethernet1-4, Ethernet10-15, Ethernet20-30. So Instead of using ```,``` we can give this flexibility of using lists.

Reply:
I understand but we don't normally make lists for accepting ranges unless the EOS command supports having multiple entries like the `groups`.

Reply:
Ok, I will adjust.